WindowsServerDocs/identity/ad-fs/operations/Configure-intranet-forms-based-authentication-for-devices-that-do-not-support-WIA.md

```diff
@@ -55,11 +55,11 @@ You can add Chrome or other user agents to the AD FS configuration that supports
 
 In AD FS configuration, add a user agent string for Chrome on Windows-based platforms:
 
-    Set-AdfsProperties -WIASupportedUserAgents ((Get-ADFSProperties | Select -ExpandProperty WIASupportedUserAgents) + "Mozilla/5.0 (Windows NT")
+    Set-AdfsProperties -WIASupportedUserAgents ((Get-ADFSProperties | Select -ExpandProperty WIASupportedUserAgents) + "Mozilla/5.0 (Windows NT"))
```

Comment:
I could be wrong, but I would expect one of the closing parentheses to be placed like this:
```suggestion
    Set-AdfsProperties -WIASupportedUserAgents ((Get-ADFSProperties | Select -ExpandProperty WIASupportedUserAgents) + "Mozilla/5.0 (Windows NT)")
```

Reply:
I tried, and you get a syntax error. We need to add Mozilla/5.0 (Windows NT) without the "".

Reply:
Oh. That is odd, are you using some extra tool providing that information? Fair enough, I will amend the suggestion above.

Reply:
Oh. Right. I forgot that this needs to be PowerShell compliant. Thanks for pointing out that detail.

Reply:
Please try again, and commit this change if it works now.

Reply:
Thanks illfated! You can commit the changes.

Reply:
I didn't notice that. Thanks for the clarification @illfated!